common/common_functions.sh

```diff
@@ -817,6 +817,135 @@ function deploy_k8s_bare {
     fi
 }
 
+function deploy_kind_cluster_with_multus {
+    local project="$1"
+    local workers_number="$2"
+
+    local status=0
+
+    deploy_kind_cluster "$project" "$workers_number"
+    let status=$status+$?
+    if [[ "$status" != "0" ]]; then
+        echo "Failed to deploy kind!!"
+        popd
+        return $status
+    fi
+
+    deploy_multus
+    let status=$status+$?
+    if [[ "$status" != "0" ]]; then
+        echo "Failed to deploy multus!!"
+        popd
+        return $status
+    fi
+}
+
+function deploy_kind_cluster {
+    local project="$1"
+    local workers_number="$2"
+
+    local status=0
+
+    ./run_kind_ci.sh --project "$project" --num-workers "$workers_number"\
```

Comment:
It looks like this call can fail after this PR  [1] will be merged 
I think it is better to discuss with @Mmduh-483 which PR should be merged first.
[1] https://github.com/Mellanox/kubernetes-ci/pull/110/files

Reply:
you are correct, phases that should not run will run now. @Mmduh-483 WDYT?

Reply:
I think we can add a flag `--phases` to run specific phases and calling this flag will ignore `--skip-phases` for simplicty

Reply:
Check PR #112

Reply:
updated the PR, can you have a look

Reply:
You need to run `prepare-ci-environment` phase also. `deploy-kind/create-cluster` sub-phase renders the config file then try to move it to the artifacts dir that created in `prepare-ci-environment`

Reply:
@Mmduh-483 I do that manually in the start script. see https://github.com/Mellanox/kubernetes-ci/pull/105/files#diff-1da590bb51c16903726785c833421c5daa35025cdc36c923aca64459aae22f1cR110